### PR TITLE
Update animals.yml

### PR DESCRIPTION
**Pull request by StarboundSolaris**
## About the PR

Disables the corrupted corgi's ability to spawn.
## Why / Balance

This has become an ongoing issue with the lighthouse map, where rarely a hostile "corrupted corgi" would spawn on roundstart and end up sending people to medical almost immediately.
## Technical details

Comments out the corrupted corgi from the random animal spawner.
## Media

N/A
## Requirements

- [ /] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ /] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: 
- remove: The corrupted corgi will no longer terrorize the crew the moment the round starts.

### CODE REVIEW

File: Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/animals.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@
       - MobGiantSpider
       - MobGiantSpiderAngry
```
Comment (deltanedas) [inline Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/animals.yml]: remove this as well

**Review by deltanedas — DISMISSED** [review]

File: Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/animals.yml (hunk 1 above)
Comment (StarboundSolaris) [reply Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/animals.yml]: Will do.

**Review by StarboundSolaris — COMMENTED** [review]
changed it.

**Comment by TadJohnson00** [comment]
Direction approved on removal of all three mobs from random spawner.

**Review by TadJohnson00 — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Just comments out things, approving.

**Comment by TadJohnson00** [comment]
@deltanedas if you approve we can merge this :blunt:

**Comment by StarboundSolaris** [comment]
I agree to allow Delta V to relicense my code from AGPL to MIT, and to remove any AGPL licensing that may be attached to said code.

**Pull request merged**